manifests/instance.pp

```diff
@@ -163,14 +163,24 @@
     path    => "/etc/redis/${redis_port}.conf",
     mode    => '0644',
     content => template('redis/redis_port.conf.erb'),
-    replace => $manage_config_file,
```

Comment:
See my above comment. The intent is for config files to update but for those not to trigger a restart unless restart_service_on_change is enabled. I think this leads to clarity on the behavior of the module. Having the module not update configs was certainly surprising to me when someone came to me asking about it. This variable is clear in its intent.